crypto/rand/drbg_lib.c

```diff
@@ -164,7 +229,14 @@ int RAND_DRBG_instantiate(RAND_DRBG *drbg,
     }
 
     drbg->state = DRBG_READY;
-    drbg->reseed_counter = 1;
+    drbg->reseed_counter = 0;
+
+    if (drbg->reseed_time != 0) {
```

Comment:
Why the if?

Reply:
It seems that when it's set to 0 it's not used. Please document that somewhere.

Reply:
I guess I would prefer that reseed_time_interval is used for that instead.

Reply:
I don't see why it's only meaningful for a non-chained DRBG. You set the interval in both cases, and when it's enabled it's always different from 0. It's just that in a chained DRBG you don't actually look at the interval value itself.

Anyway, it's fine like this.